setup.md

```diff
@@ -66,10 +66,10 @@ To install and run Flutter, your development environment must meet these minimum
    a new window:
 
    ```
-   export PATH=~/flutter/bin
+   export PATH=$PATH:$HOME/flutter/bin
```

Comment:
Why do we recommend putting flutter at the end of your path but `Android-Studio/jre/bin` at the beginning?  The usual thing is to recommend adding something to the start of the path, so I'd change this line to be analogous to line 218.